lib/ui/semantics.dart

```diff
@@ -146,6 +147,14 @@ class SemanticsAction {
   /// Accessibility focus and input focus can be held by two different nodes!
   static const SemanticsAction didLoseAccessibilityFocus = const SemanticsAction._(_kDidLoseAccessibilityFocusIndex);
 
+  /// A request that the node should be dismissed.
+  ///
+  /// A [Snackbar], for example, may have a dismiss action to indicate to the
+  /// user that it can removed after it is no longer relevant.  On Android, TalkBack
+  /// announces this after reading the label. On iOS, VoiceOver users can
+  /// perform a standard gesture to dismiss it.
+  static const SemanticsAction dismiss = const SemanticsAction._(_kDismissIndex);
```

Comment:
How does TalkBack expose the dismiss action? Is this not just a custom action of the snackbar?

Reply:
It uses https://developer.apple.com/documentation/uikit/accessibility/uiaccessibilityaction?language=objc.

Reply:
That link is for VoiceOver :)

Reply:
derp, I meant: https://developer.apple.com/documentation/objectivec/nsobject/1615091-accessibilityperformescape?language=objc

Reply:
this is what the ios MDC use for snackbar, for example